R/write_activity_tsv.R

```diff
@@ -58,7 +58,7 @@ write_activity_tsv <- function(output_dir, input_csv, compound_tsv, assay_tsv,
 
   ACTIVITY$AIDX <- input_activity$matched_AIDX
 
-  write.csv(ACTIVITY, paste(output_dir, "/ASSAY.tsv", sep =""))
+  write.csv(ACTIVITY, file.path(output_dir, "ASSAY.tsv"))
```

Comment:
The path separator is different on Windows vs Linux/Mac, which sometimes causes issues.  `file.path()` provides a more resilient way of creating the location strings regardless of what platform the user is working on.

Reply:
I think I tried to use write.table() function but I don't remember the error message I received and switched back to write.csv(). the outputs should be TSV.